backend/middleware/validate.js

```diff
@@ -0,0 +1,24 @@
+const
+  Parameter = require('parameter');
+
+let parameter = new Parameter();
+
+const validate = (req, rule) => {
```

Comment:
BTW, this is not the typical express middleware and the way you use those middlewares(including `success`/`failure`) is not the recommended paradigm.

Reply:
Thanks for that